cli/wallet/nep5.go

```diff
@@ -0,0 +1,89 @@
+package wallet
+
+import (
+	"fmt"
+
+	"github.com/nspcc-dev/neo-go/pkg/encoding/address"
+	"github.com/nspcc-dev/neo-go/pkg/rpc/client"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/urfave/cli"
+)
+
+var (
+	tokenFlag = cli.StringFlag{
+		Name:  "token",
+		Usage: "Token contract hash in LE",
```

Comment:
Can we have support for token names? Hashes are not user-friendly.

Reply:
Token name is not a unique identifier. What if multiple contracts return the same `name` and `symbol` ?
May be we can return names together with hashes when querying balances, though.

Reply:
> Token name is not a unique identifier

True, but most of the time there will be one type of XXX token in the wallet, so you can easily choose the right one.

> What if multiple contracts return the same name and symbol ?

`token XXX is ambiguous and can refer to the following hashes: AAA, BBB; please specify the hash you need`

>  May be we can return names together with hashes when querying balances, though.

I think we absolutely do need this, users are used to seeing these names at various places.

Reply:
Fixed.